Objects/EventItem/Chicken.gd

```diff
@@ -0,0 +1,13 @@
+extends "res://Objects/EventItem/EventItem.gd"
+
+var chicken_speed: float = -100
```

Comment:
This variable is unneaded, because there is move_speed var, that is initialized via editor (`EventItem.gd` line 4)

Reply:
yeah, i did not know